### PR TITLE
unexports

### PR DESCRIPTION
**Pull request by anandijain**
not sure if you wanted to get rid of `Model` too, happy to revert that one

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -1,6 +1,7 @@
 
 using Test, SHA, SparseArrays
 using SBML
+using SBML: Model, Reaction, Species
```
Comment (anandijain) [inline test/runtests.jl:4]: I could also just change the test to say SBML.Model, but for now this is probably fine?

File: docs/src/index.md [hunk 2]
```diff
@@ -1,81 +1,81 @@
-
```
Comment (exaexa) [inline docs/src/index.md:1]: what happened to this file that the diff replaces _all_ lines?

File: src/SBML.jl [hunk 3]
```diff
@@ -12,6 +12,6 @@ include("utils.jl")
 sbml = (sym::Symbol) -> dlsym(SBML_jll.libsbml_handle, sym)
 
 export SBMLVersion,
-    readSBML, Model, UnitPart, Species, Reaction, getS, getLBs, getUBs, getOCs
+    readSBML, UnitPart, getS, getLBs, getUBs, getOCs
```
Comment (exaexa) [inline src/SBML.jl:15]: cool. Honestly I think UnitPart can get removed too.

File: test/runtests.jl (hunk 1 above)
Comment (exaexa) [reply test/runtests.jl:4]: yeah, this here is better.

File: src/utils.jl [hunk 4]
```diff
@@ -1,54 +1,54 @@
...
-end
-
-"""
-    function getOCs(m::Model)::Vector{Float64}
-
-Extract the vector of objective coefficients of each reaction.
-"""
-function getOCs(m::Model)::Vector{Float64}
-    return broadcast(x -> x.oc, values(m.reactions))
-end
```
Comment (exaexa) [inline src/utils.jl:54]: also full replace instead of diff here?

**Review by exaexa — CHANGES_REQUESTED** [review]

File: test/runtests.jl (hunk 1 above)
Comment (anandijain) [reply test/runtests.jl:4]: so this is resolved, right

File: docs/src/index.md (hunk 2 above)
Comment (anandijain) [reply docs/src/index.md:1]: yeah I don't know, I was concerned to see that too. 
Any ideas?

File: src/SBML.jl (hunk 3 above)
Comment (anandijain) [reply src/SBML.jl]: think i got this

File: src/utils.jl (hunk 4 above)
Comment (anandijain) [reply src/utils.jl:54]: yes same issue, i simply did crtl f and replace all to change them.

Not sure how to prevent it

**Comment by exaexa** [comment]
The code was transplanted to #50 to avoid CRLF/LF reformat, all merged now. Thanks a lot!

**Pull request closed without merging**